admin/users/views.py

```diff
@@ -0,0 +1,40 @@
+from django.shortcuts import redirect
+
+from website.project.model import User
+
+from admin.base.views import GuidFormView, GuidView
+from admin.users.templatetags.user_extras import reverse_user
+from .serializers import serialize_user
+
+
+def disable_user(request, guid):
+    user = User.load(guid)
+    # user.disable_account()
+    user.is_disabled = True  # TODO: change to user.disable_account() after local tests
```

Comment:
This `TODO` should be done. Tests may require mocking some `mailchimp_utils`, I'm not sure if our test suite handles that.